packages/console/src/App.tsx

```diff
@@ -39,12 +40,15 @@ const Main = () => {
           <Content>
             <Routes>
               <Route path="api-resources" element={<ApiResources />} />
-              <Route path="connectors" element={<Connectors />} />
-              <Route path="connectors/:connectorId" element={<Connector />} />
               <Route path="applications">
                 <Route index element={<Applications />} />
                 <Route path=":id" element={<ApplicationDetails />} />
               </Route>
+              <Route path="connectors">
+                <Route index element={<Connectors />} />
+                <Route path="social" element={<SocialConnectors />} />
```

Comment:
will it be better to use one `<Connectors>` component, and inside this component, use `useParams()`to figure out what tab to display?

Reply:
the 2 tabs are different, they share little code, so seperate them into 2 files to be more clear?

Reply:
sry, after reconsideration, one `<Connectors>` would be better.

Reply:
cool